addons/mrp/models/mrp_routing.py

```diff
@@ -105,6 +105,20 @@ def _check_no_cyclic_dependencies(self):
         if not self._check_m2m_recursion('blocked_by_operation_ids'):
             raise ValidationError(_("You cannot create cyclic dependency."))
 
+    @api.constrains('active')
+    def _check_active(self):
+        for operation in self:
```

Comment:
```suggestion
        for operation in self:
            if operation.active:
                continue
```

Reply:
@yelizariev Changes have been applied!

Thank You!